generated/nidaqmx/_library_interpreter.py

```diff
@@ -390,6 +391,20 @@ def cfg_samp_clk_timing(
             task, source, rate, active_edge, sample_mode, samps_per_chan)
         self.check_for_error(error_code)
 
+    def cfg_time_start_trig(self, task, when, timescale):
+        cfunc = lib_importer.windll.DAQmxCfgTimeStartTrig
+        when = LibTimestamp.from_datetime(when)
+        if cfunc.argtypes is None:
+            with cfunc.arglock:
+                if cfunc.argtypes is None:
+                    cfunc.argtypes = [
+                        lib_importer.task_handle, ctypes.CVIAbsoluteTime,
```

Comment:
I'm confused by `ctypes.CVIAbsoluteTime`, where is that defined? `_lib_time.AbsoluteTime` is a ctypes type.